tests/test_consumer.py

```diff
@@ -224,42 +224,47 @@ def test_full_consume_cips_msg(self, mock_get_session):
         all_expected_data = {}
 
         all_expected_data = {
-            'data': {
-                'component': 'setup-2.8.71-7.el7_4',
-                'brew_task_id': '15477983',
-                'category': 'sanity',
-                'item': 'setup-2.8.71-7.el7_4',
-                'scratch': True,
-                'build_type': 'brew-build',
-                'issuer': 'jenkins/domain.local',
-                'rebuild': ('https://domain.local/job/ci-package-sanity-development'
-                            '/label=ose-slave-tps,provision_arch=x86_64/1835//'
-                            'rebuild/parametrized'),
-                'log': ('https://domain.local/job/ci-package-sanity-development'
-                        '/label=ose-slave-tps,provision_arch=x86_64/1835//console'),
-                'system_os': 'rhel-7.4-server-x86_64-updated',
-                'system_provider': 'openstack',
-                'ci_name': 'RPM Factory',
-                'ci_url': 'https://domain.local',
-                'ci_environment': 'production',
-                'ci_team': 'rpm-factory',
-                'ci_irc': '#rpm-factory',
-                'ci_email': 'nobody@redhat.com'
+            u'data': {
```

Comment:
bike-shedding: I'd prefer importing `unicode_literals` from `future` instead of have these each defined individually.

If we go with this, can we use this across all files?